src/chapter_1.md

```diff
@@ -1 +1,246 @@
 # Chapter 1
+
+## Create an Alumet plugin step by step
+
+The best way to get a good understanding of how Alumet's Plugin works is to do it yourself. So this chapter will create an input plugin, which
+read a random byte from a file.
+
+### Create the plugin
+
+In order to create the plugin, we need to initialize a new library. So first, go to the root directory of Alumet. You should have different
+folders containing the different plugins:
+
+```bash
+.
+├── Cargo.lock
+├── Cargo.toml
+├── LICENSE
+├── LICENSE.fr.txt
+├── README.md
+├── alumet/
+├── alumet-api-dynamic/
+├── alumet-api-macros/
+├── alumet-config.toml
+├── alumet-output.csv
+├── app-agent/
+├── app-relay-collector/
+├── plugin-csv/
+├── plugin-influxdb/
+├── plugin-nvidia/
+├── plugin-oar2/
+├── plugin-perf/
+├── plugin-rapl/
+├── plugin-relay/
+├── plugin-socket-control/
+├── target/
+├── test-dynamic-plugin-c/
+├── test-dynamic-plugin-rust/
+└── test-dynamic-plugins/
+```
+
+So let's create our plugin using:
+
+```bash
+cargo init --lib my-plugin
+```
+
+Now, go to the Cargo.toml at the root and you should see this new library:
+
+```toml
+[...]
+members = [
+    "alumet",
+    [...]
```

Comment:
Can you remove this ... which transform the files changed section in an awful red file ?